Civi/CCase/SequenceListener.php

```diff
@@ -36,6 +36,7 @@ public static function onCaseChange_static(\Civi\CCase\Event\CaseChangeEvent $ev
    * @param \Civi\CCase\Event\CaseChangeEvent $event
    *
    * @throws \CiviCRM_API3_Exception
+   * @return void
```

Comment:
Yeah, I'm not attached to that. No sweat off my back if it's removed. I only really care about writing code that passes `civicrm/coder`. ;)

FWIW, this seems like a conflict between some standard (e.g. drupal/coder vs PHPStorm or civicrm/coder?) My PHPStorm insisted on having a `@return` annotation because the function includes `return;` statements.